Assets/MixedRealityToolkit/Interfaces/BoundarySystem/IMixedRealityBoundarySystem.cs

```diff
@@ -20,6 +20,7 @@ public interface IMixedRealityBoundarySystem : IMixedRealityEventSystem, IMixedR
         /// <summary>
         /// The scale (ex: World Scale) of the experience.
         /// </summary>
+        [System.Obsolete("Use MixedRealityToolkit.AppScale or use XRDevice.GetTrackingSpaceType and XRDevice.SetTrackingSpaceType instead.")]
```

Comment:
I do not believe we want to recommend obsolete Unity methods here.

Reply:
Yes, per @keveleigh comment's I was thinking of creating an abstraction component and having these all routed there